src/registerToolbar.ts

```diff
@@ -75,19 +75,19 @@ export default function registerToolbar() {
     ToolbarItemFactory.register(PreviousAsset, {
         name: "navigatePreviousAsset",
         tooltip: "Previous Asset",
-        icon: "fas fa-arrow-circle-left",
+        icon: "fas fa-arrow-circle-up",
         group: "navigation",
         type: ToolbarItemType.Action,
-        accelerator: "ArrowLeft",
+        accelerators: ["ArrowUp", "w", "W"],
```

Comment:
According to this [doc](https://www.w3.org/TR/uievents-code/#alphanumeric-section), I could use "KeyW" so this will also work for international keyboards.  However, our keyboard binding doesn't recognize "KeyW" accelerator.  @wbreza any idea?

Reply:
Agreed - Let's update the `KeyboardManager` to use the key code identified in this article vs. the way we are currently doing it.  Let me know if you need help with this.

Reply:
looks like event.code is [not yet supported in React](https://github.com/facebook/react/issues/14102), let's create a ticket to update this later when it's supported

Reply:
We are not using the React synthetic keyboard event in our implementation - we are binding directly using standard javascript event registration which has access to the "code" property.  However - after looking @ https://caniuse.com/#search=keyboardevent.code it doesn't appear that Edge supports `code` yet... argh.

Reply:
you're right event.code works.  So should we update it now even though Edge has no support or wait till Edge supports it?